examples/lighting-app/lighting-common/lighting-app.matter

```diff
@@ -2354,6 +2354,11 @@ endpoint 0 {
   }
 
   server cluster Switch {
+    emits event SwitchLatched;
+    emits event InitialPress;
```

Comment:
See https://github.com/project-chip/connectedhomeip/pull/28225#discussion_r1274195449

Reply:
Acked, remove all *Press events to align with current implementation in all-clusters-app